library/Mockery/Adapter/Phpunit/MockeryTestCase.php

```diff
@@ -20,29 +20,16 @@
 
 namespace Mockery\Adapter\Phpunit;
 
-use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
 
-abstract class MockeryTestCase extends \PHPUnit_Framework_TestCase
-{
-    protected function assertPostConditions()
+if (class_exists("\PHPUnit_Framework_TestCase")) {
```

Comment:
No need for the leading slash, infact, it just gets deleted by php, so we should remove it.